server/honeybadger/plugins.py

```diff
@@ -26,18 +28,51 @@ def get_coords_from_google(aps):
         data['lng'] = jsondata['location']['lng']
     return data
 
-def get_coords_from_uniapple(ip):
-    logger.info('Geolocating via Uniapple API.')
-    url = 'http://uniapple.net/geoip/?ip={}'.format(ip)
-    content = urllib2.urlopen(url).read()
-    logger.info('Uniapple API response:\n{}'.format(content))
+def get_external_ip(ip):
```

Comment:
I thought you were removing this because it only made sense when ran on the client? Also, did you confirm that it worked? I still can't reach the URL. The browser says it doesn't exist and dig resolves the hostname to 0.0.0.0.

Reply:
Yes, I believe this can be removed and used in the agents themselves via javascript or otherwise. I will remove that. I'll look into doing it in this pull request.

As to you not being able to access the URL, I'm not sure why. I'm able to access api.ipify.org from every method that I try. Dig, curl, and the browser all give responses. Strange.

Reply:
It's my pi-hole server blocking it. Doh. My bad.

Reply:
Fair enough. It's a nifty API. I was confused as to why you couldn't access it.

Reply:
I don't mean to flip-flop, but I just want to verify that this shouldn't be necessary. I'm wondering if this should actually be kept, in case the beacon is triggered on the same network as the server.

In cases of demo usage, I'm wondering if this would break anything, and keeping the function doesn't seem to interrupt anything. Thoughts?